redisvl/index/index.py

```diff
@@ -354,7 +354,7 @@ def from_existing(
 
         # Validate modules
         installed_modules = RedisConnectionFactory.get_modules(redis_client)
-        validate_modules(installed_modules, [{"name": "search", "ver": 20810}])
+        validate_modules(installed_modules, [{"name": "search", "ver": 21005}])
```

Comment:
Why do we need to validate the module version up to `21005` for use of the `from_existing` method? The ability to read the index info using ft.info was available as of `20810`

Reply:
blfoat16 and float16 support was in [RediSearch was added in 2.10.5](https://redis.io/docs/latest/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes/#:~:text=What%27s%20new%20in%202.10.5)

Reply:
Ah, I see that's not needed if just reading the index. I'll switch it back.